minimal-notebook/Dockerfile

```diff
@@ -9,24 +9,25 @@ USER root
 
 # Install all OS dependencies for fully functional notebook server
 RUN apt-get update && apt-get install -yq --no-install-recommends \
+    build-essential \
+    emacs \
     git \
-    vim \
+    inkscape \
     jed \
-    emacs \
-    build-essential \
-    python-dev \
-    unzip \
     libsm6 \
+    libxext-dev \
+    libxrender1 \
+    lmodern \
     pandoc \
-    texlive-latex-base \
-    texlive-latex-extra \
+    python-dev \
     texlive-fonts-extra \
     texlive-fonts-recommended \
     texlive-generic-recommended \
+    texlive-latex-base \
+    texlive-latex-extra \
     texlive-xetex \
-    lmodern \
-    libxrender1 \
-    inkscape \
+    vim \
+    unzip \
```

Comment:
Reordered alphabetically and added libxext-dev, requirement of R packages from conda-forge